### ISSUE TITLE
Server Side Pagination

### ISSUE THREAD

**Issue by Fleeck** [TahaSh/vue-paginate#47]
Hi, what if I have to request 10 records per page with 130 pages? Is there a support for such behavior?
Looking forward to your reply

**Comment by dianseptianabl** [comment]
Yes, I have same problem too. Is it possible to request the data per page to the server, instead of load all of them at once?

**Comment by afuno** [comment]
I also interested in this question. What if there are more than a thousand entries in the database table?

_But apparently the author does not hurry to respond._

**Comment by DannyFeliz** [comment]
@Fleeck @dianseptianabl @afuno 
If you want a server side pigination use this package [vue-pagination-bootstrap](https://github.com/brunoseco/vue-pagination-bootstrap)

**Comment by afuno** [comment]
@DannyFeliz Thank you, kind man :)

**Comment by mikerogne** [comment]
This would definitely be useful
